.github/workflows/summary.yml

```
@@ -0,0 +1,34 @@
+
+      - name: Run AI inference
+        id: inference
+        uses: actions/ai-inference@v1
+        with:
+          prompt: |
+            Summarize the following GitHub issue in one paragraph:
+            Title: ${{ github.event.issue.title }}
+            Body: ${{ github.event.issue.body }}
+
+      - name: Comment with AI summary
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:18: Double quote to prevent globbing and word splitting

(shellcheck)